src/Symfony/Component/Finder/Tests/GitignoreTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Finder\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Gitignore;
+
+class GitignoreTest extends TestCase
+{
+    public function testCase1()
```

Comment:
I propose to combine some testcase and use dataProviders if possible

Reply:
@OskarStark I combined all of them in one data provider, what do you think ?